packages/config/config/devices/0x0330/homemech-200X.json

```
@@ -0,0 +1,237 @@
+{
+	"manufacturer": "HomeSys",
+	"manufacturerId": "0x0330",
+	"label": [
+		{
+			// If 1.31 then we have a 2002 unit (push button)
+			"$if": "firmwareVersion === 1.31",
+			"value": "HomeMech-2002"
+		},
+		// Else 2001 (rotary)
+		"HomeMech-2001"
+	],
+	"description": [
+		{
+			// If 1.31 then we have a 2002 unit (push button)
+			"$if": "firmwareVersion === 1.31",
+			"value": "Smart switch and dimmer (push button)"
+		},
+		// Else 2001 (rotary)
+		"Smart switch and dimmer (rotary)"
+	],
+	"devices": [
+		{
+			"productType": "0x0202",
+			"productId": "0xd01c"
+		}
+	],
+	"firmwareVersion": {
+		"min": "1.0",
+		"max": "3.0"
+	},
+	"paramInformation": [
+		{
+			"#": "2",
+			"$import": "~/templates/master_template.json#state_after_power_failure_off_on_prev",
+			"label": "Save Load State Before Power Failure"
+		},
+		{
+			"#": "3",
+			"$import": "~/templates/master_template.json#base_enable_disable",
+			"label": "Send Basic Report to the Lifeline When Load State Changes",
+			"description": "When value set as 1, repower on the dimmer, it will send Basic report automatically",
```

```suggestion
			"label": "Send Basic Report When Load State Changes",
```
We try to avoid descriptions that just restate the label in a longer way.